web/src/components/product/ProductRegistrationPage.tsx

```diff
@@ -174,6 +181,81 @@ const HostnameAlert = () => {
   );
 };
 
+const Extensions = () => {
+  // just some testing data
+  // TODO: read from backend
+  const extensions = [
+    // the current HA extension data
+    {
+      identifier: "sle-ha",
+      version: "16.0",
+      arch: "x86_64",
+      isBase: false,
+      friendlyName: "SUSE Linux Enterprise High Availability Extension 16.0 x86_64 (BETA)",
+      productLine: "",
+      available: true,
+      free: false,
+      recommended: false,
+      description:
+        "SUSE Linux High Availability Extension provides mature, industry-leading open-source high-availability clustering technologies that are easy to set up and use. It can be deployed in physical and/or virtual environments, and can cluster physical servers, virtual servers, or any combination of the two to suit the needs of your business.",
+      productType: "extension",
+      shortName: "SLEHA16",
+      name: "SUSE Linux Enterprise High Availability Extension",
+      releaseStage: "beta",
+    },
+  ];
+
+  const extensionComponents = extensions.map((ext) => (
+    <DataListItem key={`${ext.identifier}-${ext.version}`}>
+      <DataListItemRow>
+        <DataListItemCells
+          dataListCells={[
+            <DataListCell key="summary">
+              <Stack hasGutter>
+                <div>
+                  {/* remove the "(BETA)" suffix, we display a Beta label instead */}
+                  <b>{ext.friendlyName.replace(/\s*\(beta\)$/i, "")}</b>{" "}
+                  {ext.releaseStage === "beta" && (
+                    <Label color="blue" isCompact>
+                      {_("Beta")}
+                    </Label>
+                  )}
+                  {ext.free && (
+                    <Label color="green" isCompact>
+                      {_("Free")}
+                    </Label>
+                  )}
+                </div>
+                <div>{ext.description}</div>
+                <Form>
```

Comment:
I’m not entirely sure about going in this direction (directly exposing the form) in the future, but as mentioned in IRC, this is good enough for a first iteration until we know more details. However, please consider the following points:

    * Add a unique ID to the form, as there could potentially be more than one form.
    * For the same reason, add an ARIA label to help users of assistive technology distinguish between different forms
    * Do not render the form when there are no fields. In these cases, please use a simple button for triggering the register action. Again, use an ARIA label for these buttons as well, since it might be difficult to distinguish one "Register" button from another when navigating them.